src/modules/core/components/ExternalLink/ExternalLink.tsx

```diff
@@ -50,6 +52,7 @@ const ExternalLink = ({
       target="_blank"
       rel="noreferrer noopener"
       title={title}
+      {...(download !== undefined ? { download } : {})}
```

Comment:
What's happening here? I don't quite get it

Reply:
Should't this use `typeof` ?

```suggestion
      {...(typeof download === 'string' ? { download } : {})}
```

Reply:
if the prop is not passed, then it's not taken into account. I prefer to have this extra check so that it doesn't screw other links.